pkg/services/folder/service.go

```diff
@@ -9,12 +9,13 @@ import (
 
 type Service interface {
 	GetFolders(ctx context.Context, user *user.SignedInUser, orgID int64, limit int64, page int64) ([]*models.Folder, error)
-	GetFolderByID(ctx context.Context, user *user.SignedInUser, id int64, orgID int64) (*models.Folder, error)
+	// GetFolderByID(ctx context.Context, user *user.SignedInUser, id int64, orgID int64) (*models.Folder, error)
+	GetFolderByID(ctx context.Context, cmd *GetFolderQuery) (*Folder, error)
```

Comment:
should i use `GetFolderQuery` as the cmd here?

Reply:
Yes indeed!